dandischema/metadata.py

```diff
@@ -300,6 +300,18 @@ def migrate(
         if "schemaKey" not in obj:
             obj["schemaKey"] = "Dandiset"
         obj["schemaVersion"] = to_version
+    if version2tuple(schema_version) < version2tuple("0.7.0"):
```

Comment:
here will need to become some future version now that 0.7.0 is out... we might want to introduce `next` branch alike to `next` in linux development, or more specifically `next-0.8.0` and position this PR against it so we could absorb meanwhile non-breaking changes

```suggestion
    if version2tuple(schema_version) < version2tuple("0.8.0"):
```